src/lib/Scenes/Artwork/Components/__tests__/CommercialInformation-tests.tsx

```diff
@@ -38,7 +38,7 @@ describe("CommercialInformation", () => {
     }
     const component = mount(
       <Theme>
-        <CommercialInformation artwork={workInEndedAuction} />
+        <CommercialInformation artwork={workInEndedAuction as any} />
```

Comment:
Ugh dont love these `as any` but this got hairy, we should ticket to clean up these mock artworks